### ISSUE TITLE
version conflict with firebase_core 3.3.0

### ISSUE THREAD

**Issue by choim4389** [Milad-Akarie/auto_route_library#2019]
```
Because firebase_core_web 2.17.4 depends on web ^0.5.1 and no versions of firebase_core_web match >2.17.4 <3.0.0, firebase_core_web ^2.17.4 requires web ^0.5.1.
And because auto_route >=9.0.0 depends on web ^1.0.0, auto_route >=9.0.0 is incompatible with firebase_core_web ^2.17.4.
And because firebase_core 3.3.0 depends on firebase_core_web ^2.17.4 and no versions of firebase_core match >3.3.0 <4.0.0, auto_route >=9.0.0 is incompatible with firebase_core ^3.3.0.
So, because pethroomcare depends on both firebase_core ^3.3.0 and auto_route ^9.1.0, version solving failed.
```

when I try to use `auto_route ^9.1.0`, It results above error.

how to resolve it?

**Comment by Almis90** [comment]
same with geolocator package which also depends on ^0.5.1, it seems like a common version that many packages use.

**Comment by Milad-Akarie** [comment]
Fixed, in 9.2.0 thanks for reporting.
